RecipeMatch/Views/DraggableViewBackground.m

```diff
@@ -91,7 +91,7 @@ - (void)setupView{
 - (DraggableView *)createDraggableViewWithDataAtIndex:(NSInteger)index{
     DraggableView *draggableView = [[DraggableView alloc]initWithFrame:CGRectMake((self.frame.size.width - CARD_WIDTH)/2, (self.frame.size.height - CARD_HEIGHT - BTN_HEIGHT)/2, CARD_WIDTH, CARD_HEIGHT)];
     draggableView.title.text = [self.recipes objectAtIndex:index][@"recipe"][@"label"];
-    draggableView.recipeId = [self.recipes objectAtIndex:index][@"recipe"][@"uri"];
+    draggableView.recipeId = [[self.recipes objectAtIndex:index][@"recipe"][@"uri"] substringFromIndex:ID_INDEX];
```

Comment:
maybe you can add a comment about api doesn't return a clear id field, and that it can only be parsed out of the uri

also, you might want to explore a way to how to find the id based on regular expression and not fixed id, cuz if they change something in the structure of the uri, it will break a lot of your functionality.
meaning, it seems the id alway appear after `#recipe_`, so maybe try to find a way on how to find the index of this string in the general uri string, and then extrapolate the id from there